docs/how-to-docs/cert-manager.md

```diff
@@ -1,18 +1,46 @@
 
-Without wishing to duplicate documentation which can be found in the [readme](https://github.com/jetstack/cert-manager/blob/master/README.md) and or official [documentation](https://cert-manager.readthedocs.io/en/latest/), the cert-manager can effectively replace two services.
+Therefore, projects still using kube-cert-manager should modify their services to start using cert-manager instead. Note that ACP will continue to support kube-cert-manager and the internal cfssl service while they are still in use, but we do recommend shifting over to cert-manager as soon as possible as aside from security fixes there will not be any more updates to these services.
+
+Without wishing to duplicate documentation which can be found in the [readme](https://github.com/jetstack/cert-manager/blob/master/README.md) and or official [documentation](https://cert-manager.io/docs/), cert-manager can effectively replace two services:
 
 - kube-cert-manager: used to acquire certificates from LetsEncrypt.
-- cfssl: an internal Cloudflare service used to generate internal certificate _(usaually to encrypt between ingress and pod)_.
+- cfssl: an internal Cloudflare service used to generate internal certificate _(usually to encrypt between ingress and pod)_.
+
+**IMPORTANT NOTE:**
+
+`cert-manager` is being upgraded from v0.8 to v0.12. In order to allow development teams to upgrade their `cert-manager` resources according to their own schedule, both v0.8 and v.12 resources will be available concurrently for a period of time.
+
+While the older version of `cert-manager` (v0.8) is still available on the ACP platform, resources managed by the newer version of cert-manager (v0.12+) can only be accessed from the API server by suffixing the resource kind with `.cert-manager.io`.
+
+For example:
+
+```
+# to access v0.12 cert-manager resources
+kubectl -n project get certificate.cert-manager.io
+kubectl -n project get orders.acme.cert-manager.io
+kubectl -n project get challenge.acme.cert-manager.io
+```
+
+```
+# to access v0.8 cert-manager resources
+kubectl -n project get certificate
+kubectl -n project get orders
+kubectl -n project get challenge
+# or
+kubectl -n project get certificate.certmanager.k8s.io
+kubectl -n project get orders.certmanager.k8s.io
+kubectl -n project get challenge.certmanager.k8s.io
+```
 
 ### **How-tos**
 
-#### **As a developer I already have a certificate from the legacy kube-cert-manger, can I migrate?**
+#### **As a developer I already have a certificate from the legacy kube-cert-manger, how do I migrate?**
 
 Migrating from the former [kube-cert-manager](https://github.com/PalmStoneGames/kube-cert-manager) over to [cert-manager](https://github.com/jetstack/cert-manager) means creating the certificate request as below and removing the annotations from the ingress. However, the safe way would be to;
```

Comment:
This doesn't seem to relate to the section further below on acquiring an external certificate. Aren't (different) annotations still required on your ingress resource? Is creating the Certificate Request below more specifically for retrieving an "internal certificate"?

Reply:
I didn't write this section, but I think it's correct, although maybe overly cautious.
The suggested approach for migrating from kube-cert-manager to cert-manager is to not manage the certificate on an ingress through ingress annotations or labels at all and instead get cert-manager to manage a new Certificate resource, with a new secret and then specify that secret as the source for the certificate in the ingress resource.
In other word, the approach is suggesting to explicitly manage a Certificate object in order to not have any issues.
Considering certificates should be valid for at least 4 weeks (other they get renewed), another simpler approach (although slightly more risky) would be to change the annotations: remove the kube-cert-manager annotations and add the new cert-manager ones.